src/calendar/header/index.tsx

```diff
@@ -37,36 +41,37 @@ export interface CalendarHeaderProps {
   hideDayNames?: boolean;
   /** Hide month navigation arrows */
   hideArrows?: boolean;
-  /** Replace default arrows with custom ones (direction can be 'left' or 'right') */
+  /** Replace default arrows with custom ones (direction = 'left' || 'right') */
```

Comment:
maybe same way as in typescript?
```suggestion
  /** Replace default arrows with custom ones (direction = 'left' | 'right') */
```